feat(bus): get rid of amqp event bus

ostridm opened the pull request:
closes #196

derevnjuk reviewed (changes requested):
The same comments and reservations apply as in sectester-net

ostridm committed:
feat(bus): get rid of amqp event bus

BREAKING CHANGE: The `sectester/bus` package, which was intended
to provide support of communication via the AMQP transport is removed.
This change simplifies the codebase by removing deprecated options and
encourages the use of more modern and supported transport methods.

closes #196
